### PR TITLE
[Section 2] Working with numbers

### PR DESCRIPTION
**Pull request by Michelle-Lohwt**
Descriptions:

Working with numbers in ruby

### CODE REVIEW

File: sec2_the-ruby-programming-language/2.3_numbers.rb [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+puts 2 <= 5 # true
+puts 5 > 2  # true
+
+# && vs and
+## (&& has higher precedence than and)
+## screening left to right
+puts 5 && 6     # 6
+puts 6 && 5     # 5
+puts 5 and 6    # 5
+puts 6 and 5    # 6
```
Comment (Michelle-Lohwt) [inline sec2_the-ruby-programming-language/2.3_numbers.rb:69]: How come the output is not the same (line 66 - 69)

File: sec2_the-ruby-programming-language/2.3_numbers.rb [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+puts true and false # true
+puts false and true # false
+
+# || vs or
+## (|| has higher precedence than or)
+## screening left to right
+puts 6 || 5 # 6
+puts 5 || 6 # 5
+puts 6 or 5 # 6
+puts 5 or 6 # 5
```
Comment (Michelle-Lohwt) [inline sec2_the-ruby-programming-language/2.3_numbers.rb:81]: How come the output is not the same (line 78 - 81)

File: sec2_the-ruby-programming-language/2.3_numbers.rb (hunk 1 above)
Comment (tlkiong) [reply sec2_the-ruby-programming-language/2.3_numbers.rb:69]: i think you referring to wrong line?

**Review by tlkiong — APPROVED** [review]
its very deep on the way ruby works. 
You can read up about it.
Just know:

1. Don't use `and` & `or`.
2. The return value of `and & or` is the left while `&& and ||` is on the right

**Pull request merged**